flair/embeddings/transformer.py

```diff
@@ -440,13 +509,17 @@ def __build_transformer_model_inputs(
                     lang_id = lang2id.get(sentence.get_language_code(), 0)
                     model_kwargs["langs"][sentence_idx : sentence_idx + part_length] = lang_id
                     sentence_idx += part_length
-        if self.token_embedding:
+
+        if "bbox" in batch_encoding:
+            model_kwargs["bbox"] = batch_encoding["bbox"].to(device, non_blocking=True)
+
+        if self.token_embedding or self.needs_manual_ocr:
```

Comment:
we check 2x for `self.token_embedding` (line 547) and `self.needs_manual_ocr` (line 549) and jointly in line 516. The part required by both are the word_ids. Can we move everything after line 526 also in the condition for `self.token_embeddings`? looks like `self.needs_manual_ocr` only needs `word_ids_list`.